### PR TITLE
Symfony 2.8

### PR DESCRIPTION
**Pull request by tijsverkoyen**

### CODE REVIEW

File: Form/Type/ButtonIconType.php [hunk 1]
```diff
@@ -14,7 +15,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder->add(
             'example',
-            'submit',
+            SubmitType::class,
```
Comment (WouterSioen) [inline Form/Type/ButtonIconType.php:18]: :-1: (als de vorige versie nog werkt)

File: Form/Type/CollectionsType.php [hunk 2]
```diff
@@ -15,32 +17,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 'languages',
-                'collection',
```
Comment (WouterSioen) [inline Form/Type/CollectionsType.php:18]: idem

File: Form/Type/DatePickerType.php [hunk 3]
```diff
@@ -18,7 +22,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 'date_example1',
-                'date',
```
Comment (WouterSioen) [inline Form/Type/DatePickerType.php:21]: idem

File: Form/Type/ButtonIconType.php (hunk 1 above)
Comment (tijsverkoyen) [inline Form/Type/ButtonIconType.php:18]: This is deprecated and will be removed in 3.0, see https://github.com/symfony/symfony/blob/master/UPGRADE-3.0.md#form

File: Form/Type/DatePickerType.php (hunk 3 above)
Comment (tijsverkoyen) [inline Form/Type/DatePickerType.php:21]: This is deprecated and will be removed in 3.0, see https://github.com/symfony/symfony/blob/master/UPGRADE-3.0.md#form

File: Form/Type/CollectionsType.php (hunk 2 above)
Comment (tijsverkoyen) [inline Form/Type/CollectionsType.php:18]: This is deprecated and will be removed in 3.0, see https://github.com/symfony/symfony/blob/master/UPGRADE-3.0.md#form

**Pull request merged**